.github/workflows/validate_pr_name.yml

```diff
@@ -0,0 +1,23 @@
+# A github action to validate the name of a pull request contains a Jira tag:
+
+name: Validate PR name
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  validate_pr_name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate PR title
+        id: validate
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const title = context.payload.pull_request.title;
+            const regex = /[A-Z]+-\d+/;
```

Comment:
I can definitely add the extra `[A-Z]` at the start... if Jira tags need at least two characters to be valid. The `[?` and `]?` at the end seem unnecessary since I'm only looking for an "exists-in" check. Do you agree?